x/vm/module.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 // consensusVersion defines the current x/evm module consensus version.
-const consensusVersion = 9
+const consensusVersion = 10
```

Comment:
Let's leave this--I think we're going to reset this for the production release since I don't think anyone is actually running module version 9.